bddl/activity_definitions/opening_windows/problem0.bddl

```diff
@@ -7,9 +7,7 @@
         agent.n.01_1 - agent.n.01
     )
     
-    (:init 
-        (closed window.n.01_1) 
```

Comment:
Same as opening doors. Even if they start close in og by default, bddl has no way of knowing that and shouldn't assume it